mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -1538,7 +1538,7 @@ def _compute_moc(latBins, nz, latCell, regionCellMask, transportZ,
     """compute meridionally integrated MOC streamfunction"""
 
     mocTop = np.zeros([np.size(latBins), nz + 1])
-    mocTop[0, range(1, nz + 1)] = transportZ.cumsum()
+    mocTop[0, nz-1:-1:-1] = - transportZ[nz-1:-1:-1].cumsum()
```

Comment:
@vanroekel and @milenaveneziani, could you double check that this makes the same change as in https://github.com/E3SM-Project/E3SM/pull/5170 (once that gets its indexing fixed)?

The bottom entry of `mocTop` is purposefully left as all zeros.  Then, we sum up (`nz-1:-1:-1`, meaning from index `nz-1` to index `0` in steps of `-1`, keeping in mind that python has a "stop" index that is beyond the last index, rather than equal to the last index, making things unnecessarily confusing) from the bottom to the top.

Reply:
@xylar I compared this form to that in the E3SM PR (with fixed indexing) in a python script I have.  the two methods were exactly the same.  so this looks good to me.